project-base/.stylelintrc

```diff
@@ -0,0 +1,34 @@
+{
+    "rules": {
+        "block-no-empty": true,
+        "block-closing-brace-empty-line-before": "never",
+        "color-no-invalid-hex": true,
+        "color-hex-case": [ "lower", {
+            "message": "Lowercase letters are easier to distinguish from numbers"
+        } ],
+        "comment-empty-line-before": [ "always", {
+            "ignore": ["stylelint-commands", "after-comment"]
+        } ],
+        "declaration-colon-space-after": "always",
+        "declaration-block-no-duplicate-properties": [ true, {
+            ignore: ["consecutive-duplicates"]
+        } ],
+        "function-calc-no-unspaced-operator": true,
+        "indentation": [ 4, {
+            "except": ["value"],
+            "message": "Please use 4 spaces for indentation."
+        }],
+        "length-zero-no-unit": true,
+        "max-empty-lines": 2,
+        "media-feature-name-case": "lower",
+        "no-extra-semicolons": true,
+        "property-blacklist": [ "widht", "marign", "pading", "blcok" ],

```

Comment:
Interesting. I do not thing spellchecking should be part of lint rules. It might be handled by IDE much better.

Reply:
For example in Atom, there is no spell checker for css files. It think it will be better to test it. It shows warning immediately.

Reply:
`blcok`? :)

Reply:
it is black list of most used typo - so if somebody make mistake it shows error immediately